src/components/communities/Toolbar.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * @author aramsey
+ *
+ * The toolbar displayed when viewing communities
+ */
+
+import React, { useState } from "react";
+
+import { build, DotMenu } from "@cyverse-de/ui-lib";
+import {
+    Button,
+    Hidden,
+    ListItemIcon,
+    ListItemText,
+    makeStyles,
+    MenuItem,
+    TextField,
+    Toolbar,
+    Typography,
+} from "@material-ui/core";
+import { Help } from "@material-ui/icons";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+
+import ids from "./ids";
+import styles from "./styles";
+import { COMMUNITY_FILTER } from "./index";
+import { useTranslation, Trans } from "i18n";
+
+import DEDialog from "../utils/DEDialog";
+
+const useStyles = makeStyles(styles);
+
+function CommunityToolbar(props) {
+    const { parentId, filter, setFilter } = props;
+    const classes = useStyles();
+    const { t } = useTranslation(["communities", "common"]);
+
+    const [helpDlgOpen, setHelpDlgOpen] = useState(false);
+
+    const onFilterChange = (newValue) => {
+        setFilter(newValue);
+    };
+
+    const toolbarId = build(parentId, ids.TOOLBAR);
+    const dotMenuId = build(toolbarId, ids.DOT_MENU);
+
+    return (
+        <Toolbar id={toolbarId} variant="dense">
+            <Autocomplete
+                id={build(toolbarId, ids.COMMUNITY_FILTER)}
+                disableClearable
+                className={classes.filter}
+                value={filter}
+                options={Object.values(COMMUNITY_FILTER)}
+                size="small"
+                onChange={(event, newValue) => {
+                    onFilterChange(newValue);
+                }}
+                getOptionLabel={(option) =>
+                    option === COMMUNITY_FILTER.ALL_COMMUNITIES
+                        ? t("allCommunities")
+                        : t("myCommunities")
+                }
+                renderInput={(params) => (
+                    <TextField
+                        {...params}
+                        label={t("view")}
+                        variant="outlined"
+                    />
+                )}
+            />
```

Comment:
Minor - we usually do these simple selections using `https://material-ui.com/components/selects/`. Any specific reason to use `AutoComplete` ?

Reply:
For simple selects, I prefer a `TextField` with `select` and `variant="outlined"` props: https://material-ui.com/components/text-fields/#select

I can understand, though, that some users would prefer to use the mouse as little as possible, so tabbing into this field and typing the filter they want may be preferable 🤷